CONTRIBUTING.md

```diff
@@ -58,6 +57,23 @@ Look at the unassigned issues, and find an issue to which you are comfortable co
 
 ### Build in a Conda Environment
 
+#### CUDA Toolkit Setup
+
+SRF uses the CUDA Toolkit. If you have multiple versions installed on your host, then some care needs to be taken in your environment.
+For example, you may see the following error when activating the srf conda environment:
+
+`Version of installed CUDA didn't match package`
+
+Some options:
+
+- Set the variable `CUDA_HOME` to the desired CUDA install
+  - This option is good for overriding the value set in `PATH` if you have multiple installs
+  - This will also get rid of the warning messages when activating conda
+
```

Comment:
```suggestion
  - This will also get rid of the warning messages when activating conda
  - Note: This must be set before calling `conda activate` and will only work for the lifetime of that shell session. For that reason, it's best to configure this in your `.bashrc` or similar configuration file.

```